src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -102,6 +128,10 @@ export class HuiCardPicker extends LitElement {
       return nothing;
     }
 
+    const suggestedCards = this._suggestedCards(this._cards);
+    const othersCards = this._otherCards(this._cards);
+    const customCardsItems = this._customCards(this._cards);
```

Comment:
Why not have `_loadCards` do this?

Personally I would make `_loadCards` a memoized function passing `lokalize` and `suggestedCards`.

Then for `_filterCards` you just combine the 3 arrays.